api/server/volume.go

```diff
@@ -598,5 +697,9 @@ func (vd *volAPI) Routes() []*Route {
 		{verb: "POST", path: snapPath("", volume.APIVersion), fn: vd.snap},
 		{verb: "GET", path: snapPath("", volume.APIVersion), fn: vd.snapEnumerate},
 		{verb: "POST", path: snapPath("/restore/{id}", volume.APIVersion), fn: vd.restore},
+		{verb: "GET", path: credsPath("/credslist", volume.APIVersion), fn: vd.credsList},
+		{verb: "POST", path: credsPath("/credscreate", volume.APIVersion), fn: vd.credsCreate},
```

Comment:
Similarly, a POST to the "/creds" can be considered simply as create, no need for "/credscreate"

Reply:
Yes. same as above.